### DISCUSSION TITLE
Source

### DISCUSSION THREAD

**Discussion by Luth22** [lncrawl/lightnovel-crawler#1451] [General]
Hello admin! Can you add a new source? Like https://pianmanga.com if that's it's okay. Thank you.

**Comment by dipu-bd** [comment]
Please create a source request in issues page: 
https://github.com/dipu-bd/lightnovel-crawler/issues/new?assignees=&labels=source&template=2_new-source.md&title=Replace+this+with+an+url

**Reply by Luth22** [reply]
Noted admin. Thank you.
